oak_attestation_verification/src/verifier.rs

```diff
@@ -465,8 +462,9 @@ fn verify_root_layer(
     endorsements: Option<&RootLayerEndorsements>,
     reference_values: &RootLayerReferenceValues,
 ) -> anyhow::Result<()> {
-    match values.report.as_ref() {
-        Some(Report::SevSnp(report_values)) => {
+    match (values.report.as_ref(), reference_values.insecure.as_ref()) {
```

Comment:
I wonder if this would read better as an early if on the insecure reference values, instead of having to combine it with each value below, but then each of them needs to additionally read the reference values of the type it actually cares about. WDYT?

Reply:
SG, done!